assets/js/atomic/blocks/product-elements/sale-badge/support.ts

```diff
@@ -27,7 +24,7 @@ export const supports = {
 			width: true,
 			__experimentalSkipSerialization: true,
 		},
-		...( hasSpacingStyleSupport() && {
+		...( __experimentalGetSpacingClassesAndStyles === 'function' && {
```

Comment:
The `tyepeof` keyword is missing:

```suggestion
		...( typeof __experimentalGetSpacingClassesAndStyles === 'function' && {
```